src/plugins/intel_gpu/src/graph/impls/ocl/custom_primitive.cpp

```diff
@@ -50,11 +48,14 @@ struct custom_gpu_primitive_impl : typed_primitive_impl<custom_gpu_primitive> {
                              std::shared_ptr<kernel_selector::cl_kernel_data>& cl_kernel)
         : cl_kernel(cl_kernel)
         , _kernels() {
-        _kernel_id = arg.get_program().add_kernel(cl_kernel->code.kernelString);
+        auto _params = arg.get_kernel_impl_params();
+        arg.get_program().add_kernel(*_params, cl_kernel->code.kernelString);
```

Comment:
Why should we do add_kernel here? 
Can't custom primitives be handled like others, 
1) Set kernel string to impl here 
2) Before build_all : add_kernel

Reply:
Fixed it.